src/planner/mod.rs

```diff
@@ -395,6 +404,7 @@ impl HasExpectedErrors for PlannerError {
             PlannerError::Plist(_) => None,
             PlannerError::Sysctl(_) => None,
             this @ PlannerError::RosettaDetected => Some(Box::new(this)),
+            PlannerError::OsRelase(_) => None,
```

Comment:
...and here  :P

Reply:
YEah I saw the CI fail, was just building locally to make sure this time...